src/Public/Redis.ps1

```diff
@@ -238,11 +272,26 @@ function Add-RedisKey
 
         [Parameter()]
         [timespan]
-        $TTL
+        $TTL,
+
+        [Parameter()]
+        $ConnectionName,
+
+        [switch]
+        $HashType
```

Comment:
In `Get-RedisKeyDetails` we use:

```powershell
[Parameter()]
[ValidateSet('hash', 'set', 'string')]
[string]
$Type
```

Possible worth doing the same here, with just `hash` and `string` - and defaulted to `string`? (in case we get any more types)

Reply:
Done